polyglot.js

```diff
@@ -96,7 +90,7 @@ class PolyGlot {
         for (let i = messages.length - 1; i >= 0; i--) {
             let message = messages[i]
             if (message.data.type == CONST.CHAT_MESSAGE_TYPES.IC) {
-                let lang = message.getFlag("polyglot", "language") || ""
+                let lang = message.data.flags.polyglot.language || ""
```

Comment:
Yeah, I changed it because I was having issues with getFlag on "Ose" system, later on I discovered it could be my foundry was corrupted. I force-updated and it solved. Sorry about that.